PFR/neg_xlogx.lean

```diff
@@ -132,10 +132,8 @@ lemma strictConcaveOn_negIdMulLog : StrictConcaveOn ℝ (Set.Ici (0 : ℝ)) negI
 
 lemma sum_negIdMulLog_le {S : Type*} [Fintype S] {w : S → ℝ} {p : S → ℝ} (h0 : ∀ s, 0 ≤ w s)
     (h1 : ∑ s, w s = 1) (hmem : ∀ s, 0 ≤ p s) :
-    ∑ s, (w s) * negIdMulLog (p s) ≤ negIdMulLog (∑ s, (w s) * (p s)) := by
-  refine ConcaveOn.le_map_sum concaveOn_negIdMulLog ?_ h1 ?_
-  · simp [h0]
-  · simp [hmem]
+    ∑ s, (w s) * negIdMulLog (p s) ≤ negIdMulLog (∑ s, (w s) * (p s)) :=
+  ConcaveOn.le_map_sum concaveOn_negIdMulLog (by simp [h0]) h1 (by simp [hmem])
```

Comment:
This is one of the two changes I agree with.

Reply:
OK.